core/dbt/task/deps.py

```diff
@@ -45,22 +47,27 @@ def __init__(
         super().__init__(args=args, config=None, project=project)
         self.cli_vars = cli_vars
 
-    def track_package_install(self, package_name: str, source_type: str, version: str) -> None:
+    def track_package_install(
+        self, package_name: str, source_type: str, version: Optional[str]
+    ) -> None:
         # Hub packages do not need to be hashed, as they are public
-        # Use the string 'local' for local package versions
         if source_type == "local":
             package_name = dbt.utils.md5(package_name)
             version = "local"
+        elif source_type == "tarball":
+            package_name = dbt.utils.md5(package_name)
+            version = "tarball"
         elif source_type != "hub":
             package_name = dbt.utils.md5(package_name)
             version = dbt.utils.md5(version)
+
         dbt.tracking.track_package_install(
-            "deps",
-            self.project.hashed_name(),
+            self.config,
+            self.config.args,
```

Comment:
It looks like this part is undoing the deps implementation on the feature branch: https://github.com/dbt-labs/dbt-core/pull/6260/files#diff-77c52d1632e496415c4b241f4eb7ced5e8f7c6cbe207955177b2e8c6c6a54227R58-R59

Reply:
Thanks! Updated!